app/src/Molecule.re

```diff
@@ -0,0 +1,43 @@
+open Source.Project;
```

Comment:
Not immediate action needed, but given we want to have molecules part of the design system as library - we should remove this dependency. One way is could be to be explicit about the props the `ProjectCard` needs and just enumerate those explicitly. Examples of how to use `makeProps` for it, can be found in the [componets docs](https://reasonml.github.io/reason-react/docs/en/components#reactcomponent) and [this article](https://khoanguyen.me/writing-reason-react-bindings-the-right-way/). Which will only be needed once the props grow more, for now we can just put it in the make signature.

Reply:
Removed the dependency and added named props for now.